Add changelog for NewPipe 0.22.0 (983) (#7810)
TeamNewPipe/NewPipe: fastlane/metadata/android/en-US/changelogs/983.txt

> New fast forward / rewind behaviour

Reviewer: ```suggestion
New fast forward / rewind behaviour and UI
```

Reviewer 2: @opusforlife2 somehow it looks better the other way around:
```suggestion
New fast forward / rewind UI and behavior
```

Made settings searchable
Highlight pinned comments as such
Added FSFE PeerTube instance to URL opener

> Improved error reporting and new error notification

Reviewer 3: The error notification has been added and not improved, so I think we should reflect (properly) this in the changelog:

```suggestion
Improved error reporting and add new error notification
```

> Fixed first item always played in the play queue when reloading play queue manager

Reviewer 3: Probably most of users don't know what is the play queue manager and where/when/how it is used so I'd rename this entry to:

```suggestion
Fixed first item always played in the play queue when switching quality or player type
```
